src/main/java/uk/gov/hmcts/darts/common/exception/AuditApiError.java

```diff
@@ -3,25 +3,27 @@
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.darts.audit.model.AuditErrorCode;
+import uk.gov.hmcts.darts.audit.model.AuditTitleErrors;
 
 @Getter
 @RequiredArgsConstructor
 public enum AuditApiError implements DartsApiError {
 
     FILTERS_WERE_EMPTY(
-        "100",
+        AuditErrorCode.AUDIT_FILTERS_WERE_EMPTY.getValue(),
         HttpStatus.BAD_REQUEST,
```

Comment:
As the intent of the refactoring is to define the error details in openapi such that it's self-documenting, should we also pull the desired HTTP status from there, rather than hardcoding it here?

This is a general comment applicable to all implementations of `DartsApiError`